SU-SWS/stanford_magazine
# Redesign cjw

# READY FOR REVIEW


# Summary
This provides views for the magazine landing page

# Needed By (Date)
- Soon

# Urgency


# Steps to Test

1. Check out this branch
2. Check out the redesign-cjw branch on _stanford_soe_helper_
3. Clear the cache
4. Revert the features
5. Clear the cache
6. If necessary enable _stanford_magazine_issue_views_ module
7. Verify views are enabled (See troubleshooting)
8. Verify views at: /magazine and /magazine/all

## Troubleshooting:
I'm finding that when I update these features, the views are disabled. Go to the views page at:
admin/structure/views and make sure the views:
_Stanford Magazine Articles_ and _Stanford Magazine Topics_ are enabled.


# Affected Projects or Products
- SoE

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-1619
- https://stanfordits.atlassian.net/browse/SOE-1883
- https://stanfordits.atlassian.net/browse/SOE-1884
- https://stanfordits.atlassian.net/browse/SOE-1885

- Other PRs
https://github.com/SU-SOE/stanford_soe_helper/pull/20


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
